routes/index.js

```diff
@@ -322,22 +318,22 @@ module.exports = function () {
   // ==============
   // Blog
   router.get('/blog', (req, res) => {
-    const listOfPosts = []
-    fs.readdirSync('blog/').forEach(post => {
-      const filePath = `blog/${post}`
-      const text = fs.readFileSync(filePath, 'utf8')
-      let parsedWithFM = fm(text)
-      parsedWithFM.body = utils.md.render(parsedWithFM.body)
-      parsedWithFM.attributes.date = moment(parsedWithFM.attributes.date).format('MMMM Do, YYYY')
-      parsedWithFM.attributes.authors = parsedWithFM.attributes.authors.map(author => authors[author])
-      parsedWithFM.path = `blog/${post.slice(11, -3)}`
-      listOfPosts.unshift(parsedWithFM)
-    })
-    res.render('blog.html', {
-      title: 'Home',
-      posts: listOfPosts
-    })
-  })
+   const listOfPosts = []
```

Comment:
Seems like you've broken indentation here. Also, route for blog should remain unchanged.